docs/examples/extended-header-no-sidenav-medium-footer.html

```diff
@@ -6,59 +6,59 @@
 
 <div class="usa-overlay"></div>
 
-<header class="usa-header usa-header-extended" role="banner">
+<header class="usa-header usa-header--extended" role="banner">
```

Comment:
in the docs it looks like its still `usa-header-extended`

Reply:
https://designsystem.digital.gov/components/headers/#extended

Reply:
It looks like that's the original docs site (for 1.x). On the v2 docs it's still listed as `--extended`. Just to be sure I double checked and a single `-` breaks the nav